Template/Template/SceneDelegate.swift

```diff
@@ -5,6 +5,7 @@ import UIKit
 
 // MARK: SceneDelegate
 
+/// сцен делегат
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
```suggestion
final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```
дока для системного файла не нужна